src/Core/src/Platform/Gtk/RadioButtonExtensions.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using Gtk;
+using Microsoft.Maui.Graphics;
+// using Microsoft.UI.Xaml;
+// using Microsoft.UI.Xaml.Controls;
+// using Microsoft.UI.Xaml.Media;
+// using WBrush = Microsoft.UI.Xaml.Media.Brush;
+
+namespace Microsoft.Maui.Platform
+{
+	public static class RadioButtonExtensions
+	{
+		public static void UpdateIsChecked(this RadioButton platformRadioButton, IRadioButton radioButton)
+		{
+			platformRadioButton.Active = radioButton.IsChecked;
+		}
+
+		private static readonly string[] _backgroundColorKeys =
+		{
+			"RadioButtonBackground",
+			"RadioButtonBackgroundPointerOver",
+			"RadioButtonBackgroundPressed",
+			"RadioButtonBackgroundDisabled"
+		};
+
+		[MissingMapper]
+		public static void UpdateBackground(this RadioButton platformRadioButton, IRadioButton button)
+		{
+			// if (button.Background is SolidPaint solidPaint)
```

Comment:
Could implement a solid color?

`if (button.Background is SolidPaint solidPaint)`